pkg/hubble/parser/parser.go

```diff
@@ -117,6 +337,13 @@ func (p *Parser) Decode(monitorEvent *observerTypes.MonitorEvent) (*v1.Event, er
 			flow.NodeName = monitorEvent.NodeName
 			ev.Event = flow
 			return ev, nil
+		case monitorAPI.MessageTypeAgent:
+			agentNotifyMessage, ok := payload.Message.(monitorAPI.AgentNotifyMessage)
+			if !ok {
+				return nil, errors.ErrInvalidAgentMessageType
+			}
+			ev.Event = agentNotifyMessageToProto(agentNotifyMessage)
```

Comment:
I don't feel strongly about this, but I think we could move this into its own package, i.e. `pkg/hubble/parser/agent`.

Reply:
I was kind of the edge on whether to move this into its own package. Will move to `pkg/hubble/parser/agent`.